lms/envs/common.py

```diff
@@ -590,6 +590,18 @@
     }
 }
 
+COURSE_MODE_DEFAULTS = {
+    'bulk_sku': None,
```

Comment:
I just noticed that this adds some fields to LMS ENVS that seem odd to me. Is this using the older shopping cart implementation? @caesar2164 

I can understand the desire to parameterize the course mode default but this seems like commerce parameters set in the LMS. Could you help clarify this?